grc/blocks/cpp_hb_qt_gui.workflow.yml

```diff
@@ -1,5 +1,5 @@
 id: cpp_hb_qt_gui_workflow
-description: "c++ hier block qt gui workflow"
```

Comment:
OK, while we're at it: Qt is capitalized Qt

Reply:
Thanks! Fixed